setup.py

```diff
@@ -72,7 +72,7 @@
 
 base_dir = os.path.dirname(os.path.abspath(__file__))
 
-_libtpu_version = '0.1.dev20230825'
+_libtpu_version = '0.1.dev20231009'
```

Comment:
I suspect this should be 0.1.dev20231010 in order to include the open xla commit you specified.

Reply:
done.